index.d.ts

```diff
@@ -0,0 +1 @@
+export const rules: Record<string, 0 | "off">;
```

Comment:
Without `export default`, I believe it's the same thing.

Reply:
I was copied from `@types/eslint-config-prettier`. Maybe you are right.

Reply:
So if there is alreay `@types/eslint-config-prettier`, what the benefit to maintain it here?

Reply:
@JounQin Therea are many eslint config packages published to npm. eslint-config-prettier builtin support declaration file will be convenient for other packages so that they don't neeed to install both `eslint-config-prettier` and `@types/eslint-config-prettier`

Reply:
> So if there is alreay @types/eslint-config-prettier, what the benefit to maintain it here?

I'll echo zanminkian's comments here. It is really nice for have the types be included in the main package, as it avoids having to depend on two separate packages. Especially now that the year is 2024, TypeScript has spread across the ecosystem, and this is something that a lot of users will expect out of the box. And it is a pretty low maintenance cost for you!

Would love for this PR to be merged, as I was just avoid to start a new PR to this project for the exact same thing, and was surprised to see one already.